services/ml/compilation_delegate_dnnl.cc

```diff
@@ -547,7 +549,8 @@ int32_t CompilationDelegateDnnl::AddReorder(const std::string& input_name,
 int32_t CompilationDelegateDnnl::AddFusedActivation(
```

Comment:
You may rename the function, e.g. `AddActivation`

Reply:
Sound good. I have renamed it as your suggestion.